pkgs/tools/graphics/pikchr/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation {
+  pname = "pikchr";
+  version = "unstable-2021-04-07";
+
+  src = fetchurl {
+    url = "https://pikchr.org/home/tarball/90b6d5b4a3834ff0/pikchr.tar.gz";
```

Comment:
Does upstream over any easily versioned download?

Reply:
As far as I understand, tags should appear [here](https://pikchr.org/home/taglist), but there isn't any, so no.